facade/publicendpoint.go

```diff
@@ -232,3 +232,73 @@ func (f *Facade) EnablePublicEndpointPort(ctx datastore.Context, serviceid, endp
 	glog.V(2).Infof("Service (%s) updated", svc.Name)
 	return nil
 }
+
+// Adds a vhost public endpoint to a service
+func (f *Facade) AddPublicEndpointVHost(ctx datastore.Context, serviceid, endpointName, vhostName string, isEnabled, restart bool) (*servicedefinition.VHost, error) {
```

Comment:
The `restart` argument here is not used in this method AFAIK. 

Looks like the same argument in the `AddPublicEndpointPort()` method is not used.